src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -2411,6 +2423,34 @@ public static LanguageMode fromString(String value) {
       }
       return null;
     }
+
+    public int toInteger() {
+      Preconditions.checkState(this != NO_TRANSPILE);
+      switch (this) {
+        case ECMASCRIPT6:
+        case ECMASCRIPT6_STRICT:
+        case ECMASCRIPT6_TYPED:
+          return 6;
+
+        case ECMASCRIPT5:
+        case ECMASCRIPT5_STRICT:
+          return 5;
+
+        case ECMASCRIPT3:
+        default:
+          return 3;
+      }
+    }
+
+    public static int maxLevel(LanguageMode... modes) {
```

Comment:
I removed the call - I can remove this now: https://github.com/ChadKillingsworth/closure-compiler/commit/2126d1eaf30e7007acfba9d84c14f1ab5731c70a#diff-17c34c561974efe2470d59d8ba5f4efaL1308